File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int arr[] = {-1, -1};
        for(int i=0; i<nums.length; i++){
            for(int j=1; (j<nums.length && i!=j); j++){
                if((nums[i]+nums[j])==target){
                    arr[0] = i;
                    arr[1] = j;
                    return arr;
                }
            }
        }
        return arr;
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        res = []
        for i in range(0, len(nums)):
            for j in range(0, i):
                if nums[i]+nums[j]==target:
                    res.append(i)
                    res.append(j)
                    return res
        res = [-1, -1]
        return res

        